google-cloudevent-types/src/main/java/com/google/events/cloud/audit/v1/AuditLog.java

```diff
@@ -0,0 +1,122 @@
+package com.google.events.cloud.audit.v1;
```

Comment:
These files should have the licenses

Reply:
I think it would be best to add headers in the generator rather than each library, so I filed a follow-up issue.

https://github.com/googleapis/google-cloudevents/issues/88

(No libraries include the header right now).

Reply:
License headers FR: https://github.com/googleapis/google-cloudevents/pull/89